ambassador/ambassador/config.py

```diff
@@ -1324,11 +1362,14 @@ def module_config_ambassador(self, name, amod, tmod):
         if not have_amod_tls and tmod:
             self.tls_config_helper(name, tmod, tmod)
 
+        if amod and ('tracing' in amod):
+            self.tracing_config_helper(name, amod)
+
         # After that, check for port definitions, probes, etc., and copy them in
         # as we find them.
         for key in [ 'service_port', 'admin_port', 'diag_port',
                      'liveness_probe', 'readiness_probe', 'auth_enabled',
-                     'use_proxy_proto', 'use_remote_address' ]:
+                     'use_proxy_proto', 'use_remote_address', 'tracing', ]:
```

Comment:
I feel like this is redundant with the `self.tracing_config_helper` call above -- feels like you can just remove `tracing` from the array here?